apps/console/src/features/validation/pages/validation-config-edit.tsx

```diff
@@ -201,6 +215,32 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
             return;
         }
 
+        if (
+            passwordExpiryError.response &&
+            passwordExpiryError.response.data &&
+            passwordExpiryError.response.data.description
```

Comment:
This can be simplified
```suggestion
            passwordExpiryError?.response?.data?.description
```

Reply:
Addressed